src/jesse.erl

```diff
@@ -84,25 +83,21 @@ del_schema(Key) ->
 
 %% @doc Loads schema definitions from filesystem to in-memory storage.
 %%
-%% Equivalent to `load_schemas(Path, ParserFun, ValidationFun, MakeKeyFun)'
-%% where `ValidationFun' is `fun jesse_json:is_json_object/1' and
-%% `MakeKeyFun' is `fun jesse_lib:get_schema_id/1'. In this case
-%% the key will be the value of `id' attribute from the given schemas.
+%% Equivalent to `load_schemas(Path, ParserFun, ValidationFun)'
+%% where `ValidationFun' is `fun jesse_json:is_json_object/1'.
 -spec load_schemas( Path      :: string()
                   , ParserFun :: fun((binary()) -> json_term())
                   ) -> jesse_database:update_result().
 load_schemas(Path, ParserFun) ->
   load_schemas( Path
               , ParserFun
               , fun jesse_lib:is_json_object/1
-              , fun jesse_lib:get_schema_id/1
```

Comment:
@andreineculau why are you changing arity ? we currently use arity/4 with our get_schema_id fun.
it would be great if you could maintain compatibility.